projects/openjpeg/build.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+# Modifications copyright (C) 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+export SRC="/openjpeg"
+export MAIN_LINK=""
+
+if [[ $CONFIG = "libfuzzer" ]]
+then
+    export CC="clang"
+    export CXX="clang++"
+    export CFLAGS='-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero'
+    export CXXFLAGS='-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero'
+    export LINK_FLAGS='-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero'
+    export SUFFIX="fuzz"
+fi
+
+if [[ $CONFIG = "afl" ]]
+then
+    export CC="afl-clang-fast"
+    export CXX="afl-clang-fast++"
+    export CFLAGS="-g -fsanitize=address,integer,bounds,null,undefined,float-divide-by-zero"
+    export CXXFLAGS="-g -fsanitize=address,integer,bounds,undefined,null,float-divide-by-zero"
+    export LINK_FLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
+    export SUFFIX="afl"
+fi
+
+if [[ $CONFIG = "sydr" ]]
+then
+    export CC="clang"
+    export CXX="clang++"
+    export CFLAGS="-g"
+    export CXXFLAGS="-g"
+    export LINK_FLAGS="$CFLAGS"
+    export SUFFIX="sydr"
+    export MAIN_LINK="/main.o"
+fi
+
+if [[ $CONFIG = "coverage" ]]
+then
+    export CC="clang"
+    export CXX="clang++"
+    export CFLAGS="-g -fprofile-instr-generate -fcoverage-mapping"
+    export CXXFLAGS="-g -fprofile-instr-generate -fcoverage-mapping"
+    export LINK_FLAGS="$CFLAGS"
+    export SUFFIX="cov"
+fi
+
+# Build openjpeg.
+cd /openjpeg
+rm -rf build
+mkdir build
+cd build
+cmake -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
Isn't it needed to external pass compiler and flags to `cmake`?

Reply:
no need to pass flags